### PR TITLE
UX: Move Portfolio link to its own button

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

This is an alternative to https://github.com/MetaMask/metamask-extension/pull/17452 .  The tooltip being used is problematic and the portfolio link could get its proper attention by being alongside the other callout buttons.

## TODO

- [x] Write a migration to remove `portfolio` properties from state.

## Screenshots/Screencaps

PortfolioDropdown
PortFolioButton2

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: app/scripts/migrations/078.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { cloneDeep } from 'lodash';
+
+const version = 78;
+
+/**
+ * Prior to token detection v2 the data property in tokensChainsCache was an array,
+ * in v2 we changes that to an object. In this migration we are converting the data as array to object.
+ */
```
Comment (brad-decker) [inline app/scripts/migrations/078.js:8]: Should this be updated?

File: app/scripts/migrations/078.test.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      },
+    };
+
+    const newStorage = await migration78.migrate(oldStorage);
+    expect(newStorage.meta).toStrictEqual({
+      version: 78,
+    });
+  });
+
+  it('should remove the "showPortfolioToolip" property', async () => {
```
Comment (brad-decker) [inline app/scripts/migrations/078.test.js:17]: Should probably add a test to ensure that in the absence of the showPortfolioTooltip key the state structure is the same.

File: app/scripts/migrations/078.js (hunk 1 above)
Comment (darkwing) [reply app/scripts/migrations/078.js:8]: Good catch!

**Comment by jakehaugen** [comment]
I would propose an update to the spacing logic so all buttons are evenly spaced (ignoring the label text).

Of course, we'll need to account for translations so need some logic to make sure text does not overlap. 

Not sure exactly what the technical solution would be since I'm measuring from the icon vs. the text (which it seems to be doing currently). 

We could also set the logic to look good in "most" cases, knowing some translations will not allow for even spacing. 

Screenshot 2023-02-14 at 3 43 53 PM

**Comment by darkwing** [comment]
Good shout @jakehaugen !  Inspection showed something interesting:

NewIconButtons

Only the right-most icon is centered.  I'll work on that.

**Comment by darkwing** [comment]
@jakehaugen Updated per your suggestion:

NewPortFolioButtons

**Review by digiwand — APPROVED** [review]
lgtm! tested w/ macOS x Chrome

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**